### PR TITLE
Fix minor issues and leftovers

### PR DESCRIPTION
**Pull request by kang-makes**
There are a couple of left overs in the chart that are breaking (change the labels that are immutable).

It also adds things lost during rebases and defaults needed for corner cases.

### CODE REVIEW

File: .github/workflows/lint-and-test.yaml [hunk 1]
```diff
@@ -2,6 +2,11 @@ name: Lint and test PR
 
 on: pull_request
 
+# Prevent to consume free CI minutes by not running two workflows in the CI at the same time.
+concurrency:
+  group: pr-testing
+  cancel-in-progress: true
+
```
Comment (roobre) [inline .github/workflows/lint-and-test.yaml:9]: I think this may be problematic in some cases. e.g. if we're working in parallel or if renovate happens to open a PR while our tests are running. I don't see a huge benefit from this.

**Review by roobre — COMMENTED** [review]
LGTM except for the parallel workflow thing, which I don't see a huge value on.

File: .github/workflows/lint-and-test.yaml (hunk 1 above)
Comment (kang-makes) [reply .github/workflows/lint-and-test.yaml]: And renovate opens all the PRs at the same time so they will block each other.

Good point. Removed in [#dc7fffa](https://github.com/txqueuelen/stateless-dns/pull/63/commits/dc7fffad3d2d50f6625e70e20764ec19b896da45)

**Review by roobre — APPROVED** [review]
